src/analysis/modals/CloudEnvironmentModal.test.ts

```diff
@@ -143,6 +154,64 @@ const CloudEnvironmentModalDefaultProps: any = {
   filterForTool: undefined,
 };
 
+const cromwellRunning = {
```

Comment:
you should type this test data as an app to ensure it adheres to the contract as it evolves